src/vm/eeconfig.cpp

```diff
@@ -789,10 +792,18 @@ HRESULT EEConfig::sync()
         g_fEnableARM = TRUE;
     }
 
-    int forceGCconcurrent = CLRConfig::GetConfigValue(CLRConfig::UNSUPPORTED_gcConcurrent);
-    if ((forceGCconcurrent > 0) || (forceGCconcurrent == -1 && g_IGCconcurrent))
+    bool gcConcurrentWasForced = false;
+#ifdef FEATURE_CORECLR
```

Comment:
I would prefer to keep the existing behavior. I think that we do want to be able run GCStress with Concurrent GC enabled.